src/Twee/HomeScenes/Special punishments.twee

```diff
@@ -161,17 +161,18 @@
 				<<else>>\
 					You begin to notice her abdomen pushing rhythmically against your side as her breathing grows heavier.
 					
-					@@.guardian;I wasn't expecting your butt to be... occupied like this."@@
+					@@.guardian;"I wasn't expecting your butt to be... occupied like this."@@
 				<<endif>>\
 			<<else>>\
 				She <<if _b>>discovers your butt-plug and <<endif>>smirks<<if _c>> as your dangling chastity cage brushes her tights a moment later<<endif>>.
 				
-				@@.guardian;"My my, $player.name.@@\
+				@@.guardian;"My my, $player.name.\
 				<<if _c>>\
-					@@.guardian;It is rather fitting, isn't it?"@@
+					It is rather fitting, isn't it?
 				<<else>>\
-					@@.guardian;I wasn't expecting your butt to be... occupied like this."@@
+					I wasn't expecting your butt to be... occupied like this.
 				<<endif>>\
+				"@@
```

Comment:
I always have trouble stretching markup across if statements like this, but if it works, I'm find with it.